server/utils.js

```diff
@@ -1,6 +1,9 @@
 const generateRoomId = () => {
-    stubbedRoomId = 'ABCD'
-    return stubbedRoomId
+    //stubbedRoomId = 'ABCD' commented out just for now
+    var uniqid =require('uniqid');
```

Comment:
I think this should be declared at the top of the file instead of within the function. Because this would instantiate the library during every call which would impact performance. I would also use `const` over `var` because it is a constant.